FoursquareTopTests/StubGetVenueDetailsUseCase.swift

```diff
@@ -24,4 +24,26 @@ class StubGetVenueDetailsUseCase : GetVenueDetailsUseCase {
             })
         }
     }
-}
\ No newline at end of file
+    
+    func givenNoVenue() {
+        self.venue = nil
+    }
+    
+    func givenAVenue(withAddress address: String? = nil, withStatus status: String? = nil, withRating rating: Double? = nil, withPrice price: VenuePrice? = nil) {
+        self.venue = VenueViewModel.build(
+            address: address,
+            status: status,
+            rating: rating,
+            price: price
+        )
+    }
+    
+    func givenVenueWithMenu() {
```

Comment:
why not merge givenVenueWithMenu and givenVenueWithoutMenu in givenAVenue with a menuURL param?